frontend/ui/src/components/file-manager/components/proejct-box.tsx

```
@@ -0,0 +1,320 @@
+                className={
+                  "flex cursor-pointer items-center gap-1 rounded-md p-2 font-medium text-sm transition-colors"
+                }
+                disabled={currentPage === 1}
+              >
+                <ChevronLeft className="h-4 w-4" />
+                Previous
+              </Button>
+              <Flex gap="2" align={"center"}>
+                {getPageNumbers().map((page, index) =>
+                  page === "ellipsis1" || page === "ellipsis2" ? (
+                    <span key={`ellipsis-${index}`} className="px-2">
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)